src/components/structures/LoggedInView.tsx

```diff
@@ -712,19 +724,21 @@ class LoggedInView extends React.Component<IProps, IState> {
                     <ToastContainer />
                     <div className={bodyClasses}>
                         <div className="mx_LeftPanel_outerWrapper">
-                            <LeftPanelLiveShareWarning isMinimized={this.props.collapseLhs || false} />
-                            <div className="mx_LeftPanel_wrapper">
+                            <LeftPanelLiveShareWarning
+                                isMinimized={(!useNewRoomList && this.props.collapseLhs) || false}
```

Comment:
The condition `!useNewRoomList && this.props.collapseLhs` is used 3 times, can you move it in a variable